modules/lifestreetBidAdapter.js

```diff
@@ -0,0 +1,146 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'lifestreet';
+const ADAPTER_VERSION = 'prebidJS-3.0';
+
+const urlTemplate = template`https://ads.lfstmedia.com/gate/${'adapter'}/${'slot'}?adkey=${'adkey'}&ad_size=${'ad_size'}&__location=${'location'}&__referrer=${'referrer'}&__wn=${'wn'}&__sf=${'sf'}&__fif=${'fif'}&__if=${'if'}&__stamp=${'stamp'}&__pp=1&__hb=1&_prebid_json=1&__gz=1&deferred_format=vast_2_0,vast_3_0&__hbver=${'hbver'}`;
+
+/**
+ * A helper function for template to generate string from boolean
+ */
+function boolToString(value) {
+  return value ? '1' : '0';
+}
+
+/**
+ * A helper function for template
+ */
+function isInteger(value) {
```

Comment:
you have `utils.isInteger()` available for use, so you don't have to define your own implementation, wdyt?